### PR TITLE
IBX-74: Ibexa cloud setup

### PR DESCRIPTION
**Pull request by DominikaK**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://issues.ibexa.co/browse/IBX-74
| Versions      | 3.3, master, others as well?

Note: make sure if the procedure is the same < 3.3 (does the ibexa:setup --platformsh command exist there?)

### CODE REVIEW

**Review by adamwojs — COMMENTED** [review]
The only thing which I'm missing here is links to p.sh documentation. This article is just introduction to topic and that's good, but reader should be able to jump straight to more detailed documentation e.g.  "Additiona services" should link to https://docs.platform.sh/configuration/services.html

File: docs/getting_started/install_on_ibexa_cloud.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+``` bash
+composer ibexa:setup --platformsh
+```
+
+This command adds to your project configuration files required for using Ibexa Cloud.
+
+You can adapt the configuration in two places:
+
+- `.platform.app.yaml` - main configuration
+- `.platform/services.yml` - additional services such as search engines or cache
```
Comment (mnocon) [inline docs/getting_started/install_on_ibexa_cloud.md:18]: There is also `.platform/routes.yaml`, not sure if it's worth mentioning here (it should be modified when enabling Fastly, for example).

**Review by adamwojs — APPROVED** [review]

File: docs/getting_started/install_on_ibexa_cloud.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+!!! note
+
+    During the build phase (defined in the `hooks.build` configuration), files in the project have read/write permissions (can be modified).
+    
+    During deployment (defined in the `hooks.deploy` configuration), all files in the project are read-only.
+
+### Additional services
+
+`.platform/services.yaml` contains prepared setting blocks that you can uncomment
```
Comment (dabrt) [inline docs/getting_started/install_on_ibexa_cloud.md:43]:
```suggestion
`.platform/services.yaml` contains preconfigured setting blocks that you can uncomment
```

File: docs/getting_started/install_on_ibexa_cloud.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+to enable services such as Solr or Elasticsearch, or persistent Redis session storage.
+
+For information about available services,
+see [Platform.sh documentation.](https://docs.platform.sh/configuration/services.html)
+
+If you enable any of the services, you need to uncomment the relevant relationship
```
Comment (dabrt) [inline docs/getting_started/install_on_ibexa_cloud.md:49]:
```suggestion
If you enable any of the services, you must uncomment the relevant relationship
```

File: docs/getting_started/install_on_ibexa_cloud.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+## 2. Create an account
+
+Log in to https://cloud.ibexa.co or create an account if you do not have one yet.
+
+Create a project and select its region.
+
+## 3. Prepare for hosting
+
+After the project is created, the website walks you through preparing your project for hosting.
+This includes adding an ssh key, and adding Platform.sh as a git remote.
```
Comment (dabrt) [inline docs/getting_started/install_on_ibexa_cloud.md:61]:
```suggestion
This includes adding an SSH key, and adding Platform.sh as a git remote.
```

**Review by dabrt — APPROVED** [review]
Please see my minor suggestions

File: docs/getting_started/install_on_ibexa_cloud.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
+# Install on Ibexa Cloud
+
+Ibexa Cloud enables you to host your application in the cloud using the [Platform.sh](https://platform.sh/) service.
+
+## 1. Prepare configuration files
+
+If you hadn't run the `composer ibexa:setup` command during installation, run it now:
```
Comment (DominikaK) [inline docs/getting_started/install_on_ibexa_cloud.md:7]:
```suggestion
If you didn't run the `composer ibexa:setup` command during installation, run it now:
```

File: docs/getting_started/install_on_ibexa_cloud.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+- `.platform.app.yaml` - main configuration
+- `.platform/services.yml` - additional services such as search engines or cache
+- `.platform/routes.yml` - routes to additional services, for example Fastly
+
+For details about available configuration settings,
+refer to [Platform.sh documentation.](https://docs.platform.sh/configuration/app.html)
+
+### Disk space
+
+The total disk space is defined by your Ibexa Cloud subscription level.
```
Comment (DominikaK) [inline docs/getting_started/install_on_ibexa_cloud.md:26]:
```suggestion
The total disk space depends on your Ibexa Cloud subscription level.
```

**Review by Steveb-p — APPROVED** [review]
In general, looks good.

I'm trying to remember any potential pitfalls that user can get caught by, but nothing comes to mind in particular.

**Pull request merged**